components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -1,25 +1,237 @@
 #include "ekat/ekat_assert.hpp"
 #include "physics/rrtmgp/scream_rrtmgp_interface.hpp"
 #include "physics/rrtmgp/atmosphere_radiation.hpp"
+#include "physics/rrtmgp/rrtmgp_inputs_initializer.hpp"
+#include "mo_gas_concentrations.h"
 
 namespace scream {
     RRTMGPRadiation::RRTMGPRadiation (const ekat::Comm& comm, const ekat::ParameterList& params) : m_rrtmgp_comm (comm), m_rrtmgp_params (params) {
         /*
          * Anything that can be initialized without grid information can be initialized here.
          * I.e., universal constants, options, etc.
          */
+          if (!m_rrtmgp_params.isParameter("Grid")) {
+              m_rrtmgp_params.set("Grid",std::string("SE Physics"));
+          }
+
+          m_initializer = create_field_initializer<RRTMGPInputsInitializer>();
     }  // RRTMGPRadiation::RRTMGPRadiation
 
     void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_manager) {
 
+        using namespace ekat::units;
+
+        auto kgkg = kg/kg;
+        kgkg.set_string("kg/kg");
+        auto m3 = m * m * m;
+        m3.set_string("m3");
+        auto Wm2 = W / m / m;
+        Wm2.set_string("Wm2");
+        auto nondim = m/m;  // dummy unit for non-dimensional fields
+        auto micron = m / 1000000;
+
+        auto VL  = FieldTag::VerticalLevel;
```

Comment:
You could do `using namespace ShortFieldTagsNames;`. Those short names are already available in that namespace (see `field_tag.hpp`).

Reply:
I think I will leave these for now. It was helpful for me to see what each corresponded to explicitly.